sklearn/utils/extmath.py

```diff
@@ -690,9 +775,14 @@ def _randomized_eigsh(
     effective rank. Usually, `n_components` is chosen to be greater than k
     so increasing `n_oversamples` up to `n_components` should be enough.
 
-    Strategy 'value': not implemented yet.
-    Algorithms 5.3, 5.4 and 5.5 in the Halko et al paper should provide good
-    candidates for a future implementation.
+    Strategy 'value':
+    This randomized algorithm efficiently approximates eigendecompositions
+    of Hermitian matrices by projection onto a lower-dimensional subspace using basis Q.
+    Relying on Algorithm 5.3 from Halko et al.'s article, it computes B = Q*AQ,
+    finds its eigendecomposition B = VΛV*, and forms U = QV to yield A = UΛU*
+    with bounded error. Unlike the 'module' strategy, it works efficiently with
+    non-positive semidefinite matrices, handling both positive and negative
+    eigenvalues directly.
```

Comment:
I would rather suggest to have a simple comment here : 

```suggestion
    with bounded error. Unlike the 'module' strategy, it returns the top `k` eigenvalues by decreasing value: all the positive ones first, then the negative ones if any.
```

And to add a more detailed comment in the `Strategy "module"` description:

```
Strategy 'module':
    (...existing definition...)
    Unlike the 'value' strategy, this returns the top `k` eigenvalues by decreasing **module**. Therefore, when `M` is non-positive semidefinite large negative eigenvalues will be returned before small positive ones. 
    When `M` is psd both strategies lead to the same results as all eigenvalues are positive.
```